CHANGELOG.md

```diff
@@ -13,6 +13,25 @@
 
 ## Master
 
+## 1.0.0
```

Comment:
One small thing, I think is better keep this on master for now, because rocket will put 1.0.0 here when you will release :)